src/sidebar/services/annotations-exporter.ts

```diff
@@ -49,22 +56,25 @@ export class AnnotationsExporter {
 
   buildTextExportContent(
     annotations: APIAnnotationData[],
-    groupName = '',
-    /* istanbul ignore next - test seam */
-    now = new Date(),
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: TextExportOptions,
```

Comment:
Since all arguments have defaults, you could make this `Partial<TextExportOptions>` and set the default to `{}`.